DATA/UNIVERSE/SYSTEMS/Br04/Br04.ini

```diff
@@ -1363,23 +1435,14 @@ encounter = tradelane_trade_transport, 8, 0.06
 faction = co_be_grp, 0.62
 faction = co_ti_grp, 0.38
 
-[zone]
-nickname = Zone_Br04_stokes_mine_exclusion_002
-pos = 70274, 0, -16457
-shape = BOX
-size = 900, 600, 10000
-property_flags = 131072
-visit = 128
-sort = 99.5
-
 [Object]
 nickname = Br04_Trade_Lane_Ring_37
 ids_name = 260680
 pos = 34803, 0, -20002
 rotate = 0, 59, 0
 archetype = Trade_Lane_Ring
 next_ring = Br04_Trade_Lane_Ring_38
-ids_info = 461248
+ids_info = 461250

```

Comment:
Changing those IDS manually is wasted effort since the resource generator does change them again anyway.